sim/network.go

```diff
@@ -260,7 +265,7 @@ func (n *Network) Tick(adv AdversaryReceiver) (bool, error) {
 
 func (n *Network) log(level int, format string, args ...interface{}) {
 	if level <= n.traceLevel {
-		fmt.Printf("net [%.3f]: ", n.clock)
+		fmt.Printf("net [%s]: ", n.clock.Format("2006-01-02 15:04:05.000"))
```

Comment:
Please revert this to just being in seconds since the simulation started (ideally because time starts at zero)